pkgs/by-name/ch/choose-gui/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "choose-gui";
+  version = "1.3.1";
+
+  src = fetchurl {
+    url = "https://github.com/chipsenkbeil/choose/releases/download/${version}/choose";
+    hash = "sha256-HQ77LiosHWo5gwcmq3Qz2mTMIsou7Rnv1meAkMu54yU=";
+  };
+
+  dontBuild = true;
+  dontUnpack = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp ${src} $out/bin/choose
+    chmod +x $out/bin/choose
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Fuzzy matcher for OS X that uses both std{in,out} and a native GUI";
```

Comment:
`meta.changelog` is missing
https://github.com/chipsenkbeil/choose/blob/master/CHANGELOG.md

Reply:
Didn't know about this field! Thanks for the pointer, fixed on the latest commit.

Reply:
Where is the changelog?

Please, don't resolve conversation while I haven't checked.

Reply:
> Which last commit? It should be squashed...
Sorry, meant the most recent force-pushed/amended commit.

> Where is the changelog?
I must have switched machines and removed the changelog by force-pushing changes. Sorry, it's there now!